netlify.toml

```diff
@@ -56,41 +65,63 @@
     force = true
 
 [[redirects]]
-    from = "https://go.kubebuilder.io/releases/latest/:os/:arch"
+    from = "https://go.kubebuilder.io/releases/:version/:os/:arch"
+    to = "https://github.com/kubernetes-sigs/kubebuilder/releases/download/v:version/kubebuilder_:os_:arch"
+    status = 302
+    force = true
+
+# Development branch redirect.
+[[redirects]]
+    from = "https://go.kubebuilder.io/releases/master/:os/:arch"
     to = "https://storage.googleapis.com/kubebuilder-release/kubebuilder_master_:os_:arch.tar.gz"
     status = 302
     force = true
 
+# Latest redirects.
```

Comment:
The order in this file matters, righ?
Wouldn't `https://go.kubebuilder.io/releases/latest` be captured by `https://go.kubebuilder.io/releases/:version` which is placed above? Same applies for the other 2 `latest` redirects.

I think `latest` redirect need to be placed above v1. See how previously, `https://go.kubebuilder.io/releases/latest/:os/:arch` was before `https://go.kubebuilder.io/releases/:version/:os/:arch`.

Reply:
Also, https://github.com/kubernetes-sigs/kubebuilder/releases/downloads/latest/kubebuilder_linux_amd64 doesn't seem to work.